tf_model/requirements.txt

```diff
@@ -0,0 +1,4 @@
+tensorflow==2.3.1
```

Comment:
let's stick with just one `requirements.txt`. you can add scikit-learn and pandas there and we don't need this file.